pkg/apis/messaging/v1alpha1/sequence_validation_test.go

```diff
@@ -56,12 +55,8 @@ func makeValidReply(channelName string) *v1alpha1.Destination {
 
 func makeInvalidReply(channelName string) *v1alpha1.Destination {
 	return &v1alpha1.Destination{
```

Comment:
Why remove the bad namespace, wasn't that the purpose of the old function?

Reply:
Sequence was enforcing it at the yaml level, setting namespace to have minLegth = 0.
I removed the namespace here because otherwise the test fails. Destination does allow cross-namespaces... That was the discussion from last Friday. Although I could have checked here, in none of the other places these is being checked, thus I'd rather prefer consistency across objects. Does that make sense?